Packs/Hey/Scripts/Hey/Hey_test.py

```diff
@@ -0,0 +1,122 @@
+import Hey as hey
+
+
+def test_try_re():
+    # Too many extracted values -> None
+    assert hey.try_re(hey.INT_RE, '7 days and not 8 or 6', 0) is None
+
+    # Expected extracted values -> value
+    assert hey.try_re(hey.INT_RE, '7 days exactly', 0) == '7'
+    assert hey.try_re(hey.INT_RE, '7 days and not 8', 1) == '8'
+    assert hey.try_re(hey.FLOAT_RE, 'give 100.1% and not any less', 0) == '100.1'
+    assert hey.try_re(hey.BYTES_RE, 'disk size is 1337 bytes', 0) == '1337 bytes'
+
+    # No value found -> None
+    assert hey.try_re(hey.BYTES_RE, 'disk size is 1337 byte', 0) is None
+
+
+def test_try_semicolon_arg_to_dict():
+    test_str = 'a=1'
+    assert hey.semicolon_arg_to_dict(test_str) == {'a': '1'}
+
+    test_str = 'a==1'
+    assert hey.semicolon_arg_to_dict(test_str) == {'a': '=1'}
+
+    test_str = 'a_invalid,b=1'
+    assert hey.semicolon_arg_to_dict(test_str) == {'b': '1'}
```

Comment:
I think in this case we should raise an exception instead of silently ignoring. At least when dealing with args entered by the user.